api/src/main/resources/workbench.yaml

```diff
@@ -20,30 +26,36 @@ info:
     name: "BSD"
 host: "api.pmi-ops.org"
 securityDefinitions:
-  # Establish the fact that *some endpoints* are OAuth protected
-  # by defining an `aou_oauth` security mode, which we'll assing
-  # to any protected endpoints below.
+#  x-aou-note: |
```

Comment:
You can't put the extensions just anywhere, unfortunately. See https://swagger.io/docs/specification/2-0/swagger-extensions/ for details